src/datasets/arrow_dataset.py

```diff
@@ -3502,9 +3524,12 @@ def push_to_hub(
         shard_size: Optional[int] = 500 << 20,
         embed_external_files: bool = True,
     ):
-        """Pushes the dataset to the hub.
+        """Pushes the dataset to the hub as a Parquet dataset.
         The dataset is pushed using HTTP requests and does not need to have neither git or git-lfs installed.
 
+        The resulting Parquet files are self-contained: if your dataset contains :class:`Image` or :class:`Audio`
+        data, the Parquet files will store the bytes of your images or audio files.
+
```

Comment:
We should mention that this is only true if `embed_external_files` is set to True (default behavior).